internal/cli/atlas/cloudproviders/accessroles/aws/authorize.go

```diff
@@ -61,9 +61,13 @@ func (opts *AuthorizeOpts) newCloudProviderAuthorizationRequest() *atlas.CloudPr
 func AuthorizeBuilder() *cobra.Command {
 	opts := &AuthorizeOpts{}
 	cmd := &cobra.Command{
-		Use:   "authorize <ID>",
+		Use:   "authorize <roleId>",
 		Short: "Authorize an AWS IAM role.",
 		Args:  require.ExactArgs(1),
+		Annotations: map[string]string{
+			"args":     "roleId",
```

Comment:
Question: Should we use `roleID` instead of `roleId`? I feel like I've seen other flags use both caps (like --orgID) and we should be consistent if we can. (Unless the code requires it to be to be `roleId`.)

Reply:
no, we should use `roleId` for the CLI. Go lang doesn’t like `roleId` and wants `roleID` and this is why you see `ID` instead of `Id` inside the codebase but for the CLI interface we should use `Id`.

Reply:
Got it.